app/javascript/mastodon/features/explore/index.jsx

```diff
@@ -45,6 +51,10 @@ class Explore extends React.PureComponent {
     this.column = c;
   };
 
+  componentWillUnmount () {
```

Comment:
I think this lifecycle script is deprecated. Currently tracking the removal in https://github.com/mastodon/mastodon/pull/24471

Reply:
Thanks for the quick response, @nschonni! 

Hmm, I'm pretty limited when it comes to React class components 😓, any idea on which method would substitute `componentWillUnmount` in this case?